EOSBasic/Plugins/UEOS/Source/UEOS/Public/Authentication/Authentication.h

```diff
@@ -117,19 +123,19 @@ class UEOS_API UEOSAuthentication : public UObject
 	* @param UserId - Id of the user logging in
 	* @param UserToken - Credentials or token related to the user logging in
 	*/
-	UFUNCTION( BlueprintCallable, Category = "UEOS|Authentication" )
-		void						Login( ELoginMode LoginMode, FString UserId, FString UserToken );
+	UFUNCTION(BlueprintCallable, Category = "UEOS|Authentication")
```

Comment:
Keeping the consistent formatting throughout is gonna be important to maintain readability as the project grows!

Reply:
I personally am not a fan of this formatting with the large gap in between the function return type and definition of the function. Would love to come up/see a style guide when other important functionality is implemented like the OSS/Connect linkage.